### PR TITLE
Mod 9957 create link to basic usaspending api training file

### PR DESCRIPTION
**Pull request by chasls2**
**Description:**
High level description of what the PR addresses should be put here. Should be detailed enough to communicate to a PO what this PR addresses without diving into the technical nuances

**Technical details:**
The technical details for the knowledge of other developers. Any detailed caveats or specific deployment steps should be outlined here.

**Requirements for PR merge:**

1. [ ] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [x] Backend
    - [ ] Frontend <OPTIONAL>
    - [ ] Operations <OPTIONAL>
    - [ ] Domain Expert <OPTIONAL>
4. [ ] Matview impact assessment completed
5. [ ] Frontend impact assessment completed
6. [ ] Data validation completed
7. [ ] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-9957](https://federal-spending-transparency.atlassian.net/browse/DEV-9957):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/api_docs/markdown/api_tutorial.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 # Introductory Tutorial <a name="introduction"></a>
 
-Welcome to the introductory USAspending API tutorial. This tutorial is designed for people who aren't familiar with APIs and how to use them. If you already know what an _endpoint_ is and the difference between `GET` and `POST`, you'll want to visit [Endpoints](/docs/endpoints).
+Welcome to the introductory USAspending API tutorial. This tutorial is designed for people who aren't familiar with APIs and how to use them. If you already know what an _endpoint_ is and the difference between `GET` and `POST`, you'll want to visit [Endpoints](/docs/endpoints). The [Documentation INdex](https://api.usaspending.gov/docs/) provides links to introductory resources and tutorials to help you get started.
```
Comment (nick-torres) [inline usaspending_api/api_docs/markdown/api_tutorial.md:25]: INdex?

File: usaspending_api/api_docs/markdown/endpoints.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 # API Endpoints
 
-This page is intended for users who are already familiar with APIs. If you're not sure what _endpoint_ means, and what `GET` and `POST` requests are, you may find the [introductory tutorial](/docs/intro-tutorial) more useful to start.
+This page is intended as a reference for available USAspending API endpoints. The [Documentation INdex](https://api.usaspending.gov/docs/) provides links to introductory resources and tutorials to help you get started.
```
Comment (nick-torres) [inline usaspending_api/api_docs/markdown/endpoints.md:11]: INdex?

File: usaspending_api/api_docs/markdown/documentation_index.md [hunk 3]
```diff
@@ -8,7 +8,9 @@
 # Documentation Index <a name="documentation-index"></a>
 
 ## Tutorials <a name="tutorials"></a>
-* [USASpending API Introductory Tutorial](/docs/intro-tutorial) - A short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
+- [How to USe USAspending API Endpoints](https://www.youtube.com/watch?v=AEKL2LOkRZY) - A 10-minute video discussing how to use the USAspending API with couple of examples.
+  O [Side deck](https://www.usaspending.gov/data/Basic-API-Training.pdf) - with more detailed information about how to use USAspending API endpoints.
```
Comment (collinwr) [inline usaspending_api/api_docs/markdown/documentation_index.md:12]: I believe this `O` should be a `-` to properly bullet this list

Comment (collinwr) [reply usaspending_api/api_docs/markdown/documentation_index.md:12]: Also, I might change `with more` to just `More`. 

`Side Deck` should be `Slide Deck`

File: usaspending_api/api_docs/markdown/documentation_index.md [hunk 4]
```diff
@@ -8,7 +8,9 @@
...
+- [Introductory Tutorial](https://api.usaspending.gov/docs/intro-tutorial) -  short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
```
Comment (collinwr) [inline usaspending_api/api_docs/markdown/documentation_index.md:13]: Should be `A short tutorial` to be more consistent with the first bullet.

Comment (chasls2) [reply usaspending_api/api_docs/markdown/documentation_index.md]: done

File: usaspending_api/api_docs/markdown/documentation_index.md (hunk 3 above)
Comment (chasls2) [reply usaspending_api/api_docs/markdown/documentation_index.md]: done

File: usaspending_api/api_docs/markdown/documentation_index.md [hunk 5]
```diff
@@ -8,7 +8,9 @@
 # Documentation Index <a name="documentation-index"></a>
 
 ## Tutorials <a name="tutorials"></a>
-* [USASpending API Introductory Tutorial](/docs/intro-tutorial) - A short tutorial covering basic API usage, generally covering: endpoints, filtering, and autocompletes.
+- [How to USe USAspending API Endpoints](https://www.youtube.com/watch?v=AEKL2LOkRZY) - A 10-minute video discussing how to use the USAspending API with couple of examples.
```
[the remark this answers is not in the document]
Comment (chasls2) [reply usaspending_api/api_docs/markdown/documentation_index.md]: done

**Pull request merged**